src/java.base/share/classes/java/lang/reflect/AccessibleObject.java

```diff
@@ -189,6 +190,9 @@ public static void setAccessible(AccessibleObject[] array, boolean flag) {
      * <li>static final fields declared in any class or interface</li>
      * <li>final fields declared in a {@linkplain Class#isHidden() hidden class}</li>
      * <li>final fields declared in a {@linkplain Class#isRecord() record}</li>
+     * <li>final fields of a type that implements
+     * {@linkplain jdk.internal.lang.stable.TrustedFieldType}
+     * (e.g {@linkplain StableValue StableValue})</li>
```

Comment:
The API docs for a standard method can't reference a JDK internal annotation. It would be possible for the API docs to admit to an implementation specific way to do this but I think we should try to avoid this for now.

Reply:
I will remove it. Once some of the classes are public, we could add back a message of this type.